### PR TITLE
Update index state to 2022-07-01T00:00:00Z

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

**Comment by newhoggy** [comment]
Thanks @hamishmack for the fixes!

File: cardano-api/test/Test/Cardano/Api/Json.hs [hunk 1]
```diff
@@ -3,6 +3,8 @@
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE TemplateHaskell #-}
 
+{-# OPTIONS_GHC -Wno-deprecations #-} -- TODO Fix deprecations
```
Comment (Jimbo4350) [inline cardano-api/test/Test/Cardano/Api/Json.hs]: What deprecations?

File: cardano-api/test/Test/Cardano/Api/Typed/CBOR.hs [hunk 2]
```diff
@@ -191,4 +186,37 @@ roundtrip_Tx_Cddl (AnyCardanoEra era) =
 -- -----------------------------------------------------------------------------
 
 tests :: TestTree
-tests = $testGroupGenerator
+tests = testGroup "Test.Cardano.Api.Typed.CBOR"
+  [ testPropertyNamed "roundtrip witness byron CBOR"                         "roundtrip witness byron CBOR"                         prop_roundtrip_witness_byron_CBOR
```
Comment (Jimbo4350) [inline cardano-api/test/Test/Cardano/Api/Typed/CBOR.hs:190]: Can we just avoid all together using `testPropertyNamed`? It seems like a lot of additional trouble for not much benefit. Hedgehog will display the property name by default.

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]

File: cardano-api/test/Test/Cardano/Api/Json.hs (hunk 1 above)
Comment (newhoggy) [reply cardano-api/test/Test/Cardano/Api/Json.hs]: Not sure why this appeared in your review, but it's not in the PR.

File: bench/locli/src/Cardano/Analysis/Chain.hs [hunk 3]
```diff
@@ -1,6 +1,9 @@
 {-# LANGUAGE GeneralizedNewtypeDeriving#-}
-{-# OPTIONS_GHC -Wno-deprecations -Wno-orphans #-}
+
+{-# OPTIONS_GHC -Wno-orphans #-}
```
Comment (Jimbo4350) [inline bench/locli/src/Cardano/Analysis/Chain.hs:3]: :+1:

File: cardano-api/cardano-api.cabal [hunk 4]
```diff
@@ -217,7 +217,6 @@ test-suite cardano-api-test
                       , tasty
                       , tasty-hedgehog
                       , tasty-quickcheck
-                      , tasty-th
```
Comment (Jimbo4350) [inline cardano-api/cardano-api.cabal:220]: :+1: Great. The less deps we use the better.

**Review by Jimbo4350 — APPROVED** [review]
LGTM! Thanks for keeping an eye on this.

**Review by cleverca22 — APPROVED** [review]

**Review by MarcFontaine — APPROVED** [review]
Lgtm

**Review by denisshevchenko — APPROVED** [review]

**Pull request merged**